RecoHI/HiEgammaAlgos/BuildFile.xml

```diff
@@ -1,9 +1,9 @@
 <use   name="FWCore/Framework"/>
 <use   name="FWCore/ParameterSet"/>
-<use   name="DataFormats/EgammaCandidates"/>
```

Comment:
this was ["conceptually"] a correct dependency.
we do have  in RecoHI/HiEgammaAlgos/interface/TrackIsoCalculator.h
```#include "DataFormats/PatCandidates/interface/Photon.h"```
but the code is actually using ```reco::Photon```
Please revert this change and update RecoHI/HiEgammaAlgos/interface/TrackIsoCalculator.h